src/main/java/io/stargate/sgv2/jsonapi/config/constants/HttpConstants.java

```diff
@@ -15,9 +15,27 @@ public interface HttpConstants {
   /** JSON API Embedding serive Authentication token header name. */
   String EMBEDDING_AUTHENTICATION_TOKEN_HEADER_NAME = "x-embedding-api-key";
 
+  /** JSON API Embedding serive access id header name. */
+  String EMBEDDING_AUTHENTICATION_ACCESS_ID_HEADER_NAME = "x-embedding-access-Id";
+
+  /** JSON API Embedding serive secret id header name. */
+  String EMBEDDING_AUTHENTICATION_SECRET_ID_HEADER_NAME = "x-embedding-secret-Id";
+
```

Comment:
Can we make the name more aligned with the field names `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`

Reply:
Not sure about that -- if the idea is that other providers might use same 2 concepts as well?

Reply:
Not sure if any other provider will be supporting 2 key format, so didn't use AWS.

Reply:
Makes sense to me.